### PR TITLE
Add dotcms js package

### PR DESCRIPTION
**Pull request by jtesser**

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export default {
```
Comment (fmontes) [inline rollup.config.js]: I think we don't need this, right?

File: src/app/components.ts [hunk 2]
```diff
@@ -21,10 +21,10 @@ import { ServersideCondition } from './portlets/rule-engine/condition-types/serv
 import { VisitorsLocationComponent } from './portlets/rule-engine/custom-types/visitors-location/visitors-location.component';
 import { VisitorsLocationContainer } from './portlets/rule-engine/custom-types/visitors-location/visitors-location.container';
 import {SiteSelectorComponent as SiteSelectorComponentDotJS} from '../dotcms-js/components/site-selector/site-selector.component';
-import {BreadcrumbComponent} from '../dotcms-js/components/breadcrumb/breadcrumb.component';
-import {SiteTreeTableComponent} from '../dotcms-js/components/site-treetable/site-treetable.component';
-import {SiteDatatableComponent} from '../dotcms-js/components/site-datatable/site-datatable.component';
-import {TreeableDetailComponent} from '../dotcms-js/components/treeable-detail/treeable-detail.component';
+import {DotcmsBreadcrumbModule} from '../dotcms-js/components/breadcrumb/breadcrumb.component';
```
Comment (fmontes) [inline src/app/components.ts]: You have this in the app.module, I think we don't need it here.

**Comment by fmontes** [comment]
Are you updating the readme?

File: rollup.config.js (hunk 1 above)
Comment (jtesser) [inline rollup.config.js]: we do. I am using it

File: src/app/components.ts (hunk 2 above)
Comment (jtesser) [inline src/app/components.ts]: cleaned

**Pull request merged**